### DISCUSSION TITLE
Dubble text notification

### DISCUSSION THREAD

**Discussion by FlyingSquirrel03** [InfiniTimeOrg/InfiniTime#1655] [Q&A] [answered]
### Verification

- [X] I searched for similar bug reports and found none was relevant.

### What happened?

Whenever I get a text the watch displays it twice

### What should happen instead?

One notification

### Reproduction steps

Get a text

### Version

1.11

### Companion app

Gadgetbridge 0.71.2a-banglejs

**Comment by infinitytec** [comment, marked as answer]
I had this issue with GadgetBridge as well. Turned out the solution was to disable the SMS notification. My text app would then be pushing a notification as well, so that's why I would get two. I also had this with phone calls, which was more annoying.

**Reply by FlyingSquirrel03** [reply]
Ooh, thank you so much.

**Reply by infinitytec** [reply]
No problem!

**Reply by JF002** [reply]
I also noticed that sometimes, the watch would vibrate twice when receiving some notifications, but I've never tried to debug it. Thanks for figuring that out!

This could also be the solution to this issue : https://github.com/InfiniTimeOrg/InfiniTime/issues/981

So, basically, you just need to set the SMS notification in Gadgetbridge to 'never'?

**Reply by infinitytec** [reply]
Yep! That's what I've got. Or turn off notifications from your SMS app.

I use Signal for SMS. SMS messages would double but Signal ones wouldn't. Disabling SMS notifications in GB stopped the double SMS notifications.
